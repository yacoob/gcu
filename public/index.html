<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <!--[if lt IE 9]>
  <script src='https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js'></script>
  <script src='https://oss.maxcdn.com/respond/1.4.2/respond.min.js'></script>
  <![endif]-->
  <link href='/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <meta name='description' content='Welcome to tactical grace!'/>
  <link href='http://gcu.tactical-grace.net/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>tactical grace</title>
  <meta content='tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/'>tactical grace</a></h1>
</div> <!-- column -->
  <div class='col-xs-12 col-sm-10 col-sm-offset-2 col-md-4 col-md-offset-0 text-center voffset-top'>
    <form action='//www.google.com/search' role='search'>
      <div class='input-group input-group-sm'>
        <span class='input-group-addon'><span class='glyphicon glyphicon-search'></span></span>
        <input name='q' type='search' class='form-control' placeholder='Search...'>
        <input name='sitesearch' type='hidden' value='http://gcu.tactical-grace.net/'/>
      </div> <!-- input group -->
    </form>
  </div> <!-- column -->
</div> <!-- row -->
<div class='row voffset-top'>
  <div class='col-xs-12 col-sm-4 col-md-4 text-center'>
  <div class='model'>
    <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2016-11-19/s400-p/IMG_20161105_233930.jpg'>
    <div class='overlay'>
      <a href='/hg/grimoire/#2016-11-19'></a>
      <h3>Grimoire</h3>
    </div>
  </div>
</div>
  <div class='col-xs-12 col-sm-4 col-md-4 text-center'>
  <div class='model'>
    <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2016-10-16/s400-p/IMG_20161030_214738.jpg'>
    <div class='overlay'>
      <a href='/pg/00-raiser/#2016-10-16'></a>
      <h3>00 Raiser</h3>
    </div>
  </div>
</div>
  <div class='col-xs-12 col-sm-4 col-md-4 text-center'>
  <div class='model'>
    <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2016-05-02/s400-p/DSC01352.jpg'>
    <div class='overlay'>
      <a href='/mg/red-frame/#2016-05-02'></a>
      <h3>Astray Red Frame</h3>
    </div>
  </div>
</div>
</div> <!-- row -->
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='http://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
</body></html>