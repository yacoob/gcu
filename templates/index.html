{# vim: set tw=0 ft=jinja: #}
{% extends 'base.html' -%}
{% block page_vars -%}
{% set debug = True -%}
{% set navbar = False -%}
{% set searchbar = True -%}
{% endblock page_vars -%}
{% block main -%}
{% set entry_pages = [] -%}
{% for g in config.extra.grades -%}
  {% set grade = get_section(path=g ~ '/_index.md') -%}
  {% for k in grade.subsections -%}
    {% set kit = get_section(path=k) -%}
    {% for p in kit.pages -%}
      {%- set_global entry_pages = entry_pages | concat(with=p) -%}
  {% endfor -%}
  {% endfor -%}
{% endfor -%}
<div class='row voffset-top'>
{% set seen = [] -%}
{% for page in entry_pages|sort(attribute='date')|reverse -%}
  {% set kit_on_page = page.ancestors.2 -%}
  {% if seen is containing(kit_on_page) -%}
    {% continue -%}
  {% else -%}
    {{ macros::kit_cover(kit=page, width=4) }}
    {%- set_global seen = seen|concat(with=kit_on_page) -%}
  {% endif -%}
  {% if seen|length == config.extra.recent_entry_count -%}
    {% break -%}
  {% endif %}
{% endfor %}
</div> <!-- row -->
{% endblock main -%}
