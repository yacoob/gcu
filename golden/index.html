<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <link href='https://cdn.jsdelivr.net/npm/bootstrap3@3/dist/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <meta name='description' content='Welcome to tactical grace!'/>
  <link href='https://gcu.tactical-grace.net/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>tactical grace</title>
  <meta content='tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/'>tactical grace</a></h1>
</div> <!-- column -->
  <div class='col-xs-12 col-sm-10 col-sm-offset-2 col-md-4 col-md-offset-0 text-center voffset-top'>
    <form action='//www.google.com/search' role='search'>
      <div class='input-group input-group-sm'>
        <span class='input-group-addon'><span class='glyphicon glyphicon-search'></span></span>
        <input name='q' type='search' class='form-control' placeholder='Search...'>
        <input name='sitesearch' type='hidden' value='gcu.tactical-grace.net'/>
      </div> <!-- input group -->
    </form>
  </div> <!-- column -->
</div> <!-- row -->
<div class='row voffset-top'>
  <div class='col-xs-12 col-sm-4 col-md-4 text-center'>
  <div class='model'>
    <img class='img-rounded img-responsive' src='/photos/thumb/gcu-01018.jpg'>
    <div class='overlay'>
      <a href='/varia/odaiba-unicorn/#2019-05-29'></a>
      <h3>Odaiba Unicorn</h3>
    </div>
  </div>
</div>
  <div class='col-xs-12 col-sm-4 col-md-4 text-center'>
  <div class='model'>
    <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00999.jpg'>
    <div class='overlay'>
      <a href='/rg/unicorn/#2019-04-07'></a>
      <h3>Unicorn</h3>
    </div>
  </div>
</div>
  <div class='col-xs-12 col-sm-4 col-md-4 text-center'>
  <div class='model'>
    <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00981.jpg'>
    <div class='overlay'>
      <a href='/varia/formania-ex-sazabi/#2018-10-21'></a>
      <h3>Sazabi Formania Ex</h3>
    </div>
  </div>
</div>
</div> <!-- row -->
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='https://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
</body></html>